lua/carbon-now.lua

```diff
@@ -33,29 +16,34 @@ local function query_param_encode(str)
   return str
 end
 
--- helper function to encode a k,v table into encoded query params
+---helper function to encode and concatenate a [k,v] table
+---as query parameters. iEx: {a = b, c = d} --> a=b&c=d
+---@param values table
+---@return string # Concatanation of the enconded query parameters
+---@nodiscard
 local function encode_params(values)
+  ---@type table<string, any>
   local params = {}
   for k, v in pairs(values) do
-    if type(v) ~= "string" then
-      v = tostring(v)
-    end
-    table.insert(params, k .. "=" .. query_param_encode(v))
+    local parsed_value = tostring(v)
+    local encoded_value = query_param_encode(parsed_value)
```

Comment:
since we are not having any errors or error treatment here, i believe this var is unnecessary, can you just put the calculation back in the `table.insert` call please?

Reply:
You're right, i will revert it on the next commit

Reply:
Fixed on [8aa5de1](https://github.com/ellisonleao/carbon-now.nvim/pull/4/commits/8aa5de1d050c41f69bdedd20aebaff80a4f43313)